lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
I mean, the issue is whether the `SysV` structure should be able to itself fallback to a `Simple`'s functionality. That's what I did here to make that work. 
I don't see a justification for writing it in C if it's doing exactly what it's doing here, except in more verbose C.